src/index.ts

```diff
@@ -440,18 +444,18 @@ export const walletAddressToNameAndProfilePicture = async (
 ): Promise<WalletNameAndProfilePicture> => {
   const solanaPFPStandardImageURL =
     await getProfilePictureUsingSolanaPFPStandard(connection, wallet);
-  const dotAbcOrBonkOrPoor = await walletToDotAbcDotBonkOrDotPoor(
+  const dotAnything = await walletToDotAnything(
     connection,
     wallet
   );
   // .abc, .bonk and .poor service doesn't have a profile picture, so use Solana PFP Standard
```

Comment:
Update the comment. It's no longer `.abc, .bonk and .poor service` just call it `ANS Domains`

Reply:
updated